nestkernel/node.h

```diff
@@ -701,6 +708,15 @@ class Node
 
   virtual double get_LTD_value( double t );
 
+  virtual double get_leak_propagator() const;
```

Comment:
The remaining e-prop related virtual functions in the class `Node` are the ones with which we access private attributes of the different eprop neuron models from within the eprop synapse. `get_leak_propagator` for example makes `V_.P33_` of the post-synaptic neuron, which is needed to calculate the weight update, available in the synapse. These functions are all implemented in the e-prop neuron models. Does it, in this context, still make sense to move them to the `ArchivingNode`? In practice, this would mean to save them in the `ArchivingNode`. Maybe, alternatively, to reduce the number of virtual functions one could bundle all the getter functions into one general `get_node_parameters_for_eprop` with which all the relevant parameters organized in a data structure can be retrieved in one go?